mucommander-viewer-binary/src/main/java/com/mucommander/viewer/binary/BinaryEditor.java

```diff
@@ -139,8 +154,63 @@ private void initMenuBars() {
         editMenu.add(pasteMenuItem, copyMenuItemPosition + 2);
         editMenu.add(deleteMenuItem, copyMenuItemPosition + 3);
 
+        undoMenuItem = MenuToolkit.createMenuItem(
+                Translator.get("binary_editor.undo"),
+                menuItemMnemonicHelper,
+                DesktopManager.getActionShortcuts().getDefaultKeystroke(ActionType.Undo),
+                e -> {
+                    try {
```

Comment:
Move the body to a new method that catches and logs the exception. This way you could simply do sth like this: `e -> doUndo` - a cleaner code, plus you would avoid code duplication (there's another occurrence of undo code below).